css-layout-classes/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title> CSS LAYOUT CLASSES </title>
+  <link href="styles.css" type="text/css" rel="stylesheet" />
+</head>
+
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1> Choose your character! </h1>
+      </div>
+    </div>
+
+    <div class="super-smash">
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="link">
+        </div>
+        <div class="column-one-third">
+          <article>
+            <div class="row">
+              <div class="column-full">
+              </div>
+            </div>
+            <div class="row">
+              <div class="column-one-half">
+                <h2> Super Smash Bros. Ultimate </h2>
+              </div>
+            </div>
+            <div class="row">
+              <div class="column-one-half">
+                <p>
+                  Super Smash Bros. Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely criticized.
+                </p>
+              </div>
```

Comment:
Same note above about moving this outside of the current `column-one-third` `div` it's currently in. ^